packages/nodes-base/nodes/KoboToolbox/descriptions/Operations.ts

```diff
@@ -0,0 +1,130 @@
+import {
+	INodeProperties,
+} from 'n8n-workflow';
+
+export const operations = [
+			{
+				displayName: 'Operation',
+				name: 'operation',
+				type: 'options',
+				displayOptions: {
+					show: {
+						resource: [
+							'form',
+						],
+					},
+				},
+				options: [
+					{
+						name: 'Get',
+						value: 'get',
+						description: 'Retrieve a form definition',
+					},
+				],
+				default: 'get',
+				description: 'The operation to perform.',
+			},
+			{
+				displayName: 'Operation',
+				name: 'operation',
+				type: 'options',
+				displayOptions: {
+					show: {
+						resource: [
+							'submission',
+						],
+					},
+				},
+				options: [
+					{
+						name: 'Query',
```

Comment:
Rename to Get All

Reply:
- Query operation is missing pagination
- Add two collections. One is called filters and the other one is called options. Filters will have only the query parameter. And, options will have all the other parameters.